bears/csv/CSVLintBear.py

```diff
@@ -1,11 +1,33 @@
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
 @linter(executable='csvlint',
         output_format='regex',
-        output_regex=r'\d\. (?P<message>.+(s|g|e|w|d)\.*)'
-                     r'( |$)(?P<additional_info>.*)')
+        output_regex=r'\n[0-9]+\.\s(?P<origin>(?P<severity>\w+))\.\s'
```

Comment:
The outputs start with the first line as INVALID.
The errors start from next line. 
I'll fix the \d+